client/components/common/tce-core/ElementList.vue

```diff
@@ -48,6 +48,9 @@ import Draggable from 'vuedraggable';
 import get from 'lodash/get';
 import { getElementId } from 'tce-core/utils';
 import last from 'lodash/last';
+import { mapChannels } from '@/plugins/radio';
+
+const CE_FOCUS_EVENT = 'element:focus';
```

Comment:
Not an action item for now since it might result in some bigger conflicts, but at some point, we should start extracting these to have a single listing/overview of used events and the ability to easily change.